app/bracket/create/page.js

```
@@ -3,4 +1,300 @@
+  const [error, setError] = useState("");
+  const router = useRouter();
+
+  const handleInputChange = (e) => {
+    setTeams(e.target.value);
+    setError("");
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+
+    if (isNaN(teams) || teams === "") {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)